### ISSUE TITLE
Exclusion option not working for Sonarr

### ISSUE THREAD

**Issue by KaiStarkk** [excludarr/excludarr#25]
The `--exclusion` option works in Radarr, but is not working in Sonarr.

**Comment by KaiStarkk** [comment]
Appears to be an upstream issue with Pyarr: https://github.com/totaldebug/pyarr/issues/175

**Comment by barsa2000** [comment]
Yeah, that library has a lot of problems, I had to monkey patch it a couple of times for [arr_setup](https://github.com/excludarr/excludarr/blob/5fb26a77baa2c9c4786d1192ac98a0675a6733bc/arr-setup/arr_setup/utils/patch_pyarr.py), I could monkey patch here too

**Comment by Amateur-God** [comment]
has this been fixed? i was about to rewrite the excludarr myself when i stumbled across the issue with this repo linked
